src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs

```diff
@@ -47,12 +53,27 @@ public interface IQueryIncludeBuilder<T> : IIncludeBuilder<T, IQueryIncludeBuild
         IQueryIncludeBuilder<T> IncludeCounters(Expression<Func<T, string>> path, string[] names);
 
         IQueryIncludeBuilder<T> IncludeAllCounters(Expression<Func<T, string>> path);
+
+        IQueryIncludeBuilder<T> IncludeTimeSeries(Expression<Func<T, string>> path, string name, DateTime from, DateTime to);
+
     }
 
     public class IncludeBuilder
     {
         internal HashSet<string> DocumentsToInclude;
 
+        internal IEnumerable<TimeSeriesRange> TimeSeriesToInclude
+        {
+            get
+            {
+                if (TimeSeriesToIncludeBySourcePath == null)
+                    return null;
+
+                TimeSeriesToIncludeBySourcePath.TryGetValue(string.Empty, out var value);
```

Comment:
Why string empty here?

Reply:
include time series of original document